core/src/main/java/com/backblaze/b2/client/structures/B2DownloadByIdRequest.java

```diff
@@ -67,13 +112,45 @@ public Builder setRange(B2ByteRange range) {
         }
 
         public B2DownloadByIdRequest build() {
-            return new B2DownloadByIdRequest(fileId, range, b2ContentDisposition);
+            return new B2DownloadByIdRequest(fileId,
+                    range,
+                    b2ContentDisposition,
+                    b2ContentLanguage,
+                    b2Expires,
+                    b2CacheControl,
+                    b2ContentEncoding,
+                    b2ContentType);
         }
 
         @SuppressWarnings("unused")
         public Builder setB2ContentDisposition(String b2ContentDisposition) {
             this.b2ContentDisposition = b2ContentDisposition;
             return this;
         }
+
+        public Builder setB2ContentLanguage(String b2ContentLanguage) {
+            this.b2ContentLanguage = b2ContentLanguage;
+            return this;
+        }
+
+        public Builder setB2Expires(String b2Expires) {
```

Comment:
it might be nice to provide a convenience version of this that takes a LocalDateTime object, assumed to be in UTC.